examples/vanilla/src/client.js

```diff
@@ -0,0 +1,63 @@
+import * as Y from 'yjs'
+import { createYjsProvider } from '@y-sweet/client'
+import { createColorGridUI } from './ColorGrid'
+
+const COLORS = ['#500724', '#831843', '#9d174d', '#be185d', '#db2777', '#f472b6', '#f9a8d4', null]
+const GRID_SIZE = 10
+const QUERY_PARAM = 'doc'
+
+async function main() {
+  // First, fetch a client token that can access the docId in the URL.
+  // Or, if the URL does not contain a docId, get a client token for a new doc.
+  const url = new URL('http://localhost:8080/client-token')
```

Comment:
Should this be :3000? As I understand it, `/client-token` refers to a path on the server served by vite (via `vite src --port 3000`) rather than y-sweet server, which runs on 8080 by default.

Reply:
This is correct because this demo's server runs on 8080. (Vite only runs a dev server for rebuilding / live-reloading on changes.) I forgot that the y-sweet server runs on 8080 by default, though. Maybe I should change this to 9090 so it doesn't conflict with the y-sweet server when folks choose to run one with these demos.

Reply:
Ah yeah, I ran into that when trying to run it locally.

I think running on 9090 makes sense, so it would be:

Static -> :3000
Express.js server -> :9090
y-sweet -> :8080

Reply:
Actually, weirdly it works for me when I run both servers (express.js and y-sweet) on :8080 (!)

but I think the change is good because I don't know why it works for me